NwPluginAPI/Core/Server.cs

```diff
@@ -83,6 +92,48 @@ public static int ReservedSlots
 			set => CustomNetworkManager.reservedSlots = value;
 		}
 
+		/// <summary>
+		/// Get server <see cref="PermissionsHandler"/>
+		/// </summary>
+		public static PermissionsHandler PermissionsHandler => ServerStatic.GetPermissionsHandler();
+
+		/// <summary>
+		/// Get actual ticks per second of the server.
+		/// </summary>
+		public static double Tps => Math.Round(1f / Time.smoothDeltaTime);
+
+		/// <summary>
+		/// Get actual frames of the server.
+		/// </summary>
+		public static double Frames => Math.Round(1f / Time.deltaTime);
+
+		/// <summary>
+		/// Get or set server spawn protection time.
+		/// </summary>
+		public static float SpawnProtectTime
+		{
+			get => SpawnProtected.SpawnDuration;
+			set => SpawnProtected.SpawnDuration = value;
+		}
+
+		/// <summary>
+		/// Get or set if server is heavily modded.
+		/// </summary>
+		public static bool IsHeavilyModded
+		{
+			get => CustomNetworkManager.HeavilyModded;
+			set => CustomNetworkManager.HeavilyModded = value;
+		}
+
+		/// <summary>
+		/// Get or set server has whitelist enabled.
+		/// </summary>
+		public static bool IsWhitelisted
```

Comment:
This can be confusing, because it doesn't control whether whitelist is enabled. It only controls whether server should report itself as using ANY whitelisting solution to the Central Servers.

Reply:
ok i will remove it